MapboxCoreNavigation/RouteController.swift

```diff
@@ -165,6 +159,62 @@ open class RouteController: NSObject {
         locationManager.stopUpdatingLocation()
         locationManager.stopUpdatingHeading()
     }
+    
+    /**
+     The most recently received user location.
+     
+     This is a raw location received from `locationManager`. To obtain an idealized location, use the `snappedLocation` property.
+     */
+    public var location: CLLocation?
```

Comment:
Do you think this should be public? If a developer needs the raw user location, shouldn't they just create their own location manager?